src/reaper_osara.cpp

```diff
@@ -3713,11 +3713,18 @@ void moveToItem(int direction, bool clearSelection=true, bool select=true) {
 		if (take) {
 			s << " " << GetTakeName(take);
 		}
-		int takeCount = CountTakes(item);
-		if (takeCount > 1) {
+		int totalTakes = CountTakes(item);
+		int nonEmptyTakes = 0;
+		for (int i = 0; i < totalTakes; ++i) {
```

Comment:
nit: Let's use `t` instead of `i` here to be a bit more descriptive: we're dealing with takes.